app/views/content_items/topic.html.erb

```diff
@@ -26,9 +26,24 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <% if @content_item.display_as_accordion? %>
-        <%= render "govuk_publishing_components/components/accordion", {
-          items: @content_item.accordion_content
-        } %>
+        <div data-module="gtm-click-tracking">
```

Comment:
`gtm-click-tracking` can be applied directly to the component as a `module` option in `data_attributes`.